dependencies/graknlabs/dependencies.bzl

```diff
@@ -62,8 +62,8 @@ def graknlabs_simulation():
 def graknlabs_verification():
     git_repository(
         name = "graknlabs_verification",
-        remote = "https://github.com/graknlabs/verification",
-        commit = "15e9132c9cc5d173edd81afe22ac60d04a1abe02",  # sync-marker: do not remove this comment, this is used for sync-dependencies by @graknlabs_verification
+        remote = "https://github.com/alexjpwalker/verification",
```

Comment:
need to revert this before merging

Reply:
This will be reverted in a separate PR later - it's tricky to remove at the moment because the PRs have got a bit mixed up.